examples/deployments/uc_functions/run.py

```diff
@@ -0,0 +1,141 @@
+"""
```

Comment:
Can we add a README.md to this examples section? It wasn't obvious that I need to install the databricks SDK to use this. Got a failure when launching deployments server:

```
  File "/Users/corey.zumar/mlflow/mlflow/gateway/providers/__init__.py", line 17, in get_provider
    from mlflow.gateway.providers.openai import OpenAIProvider
  File "/Users/corey.zumar/mlflow/mlflow/gateway/providers/openai.py", line 13, in <module>
    from mlflow.gateway.uc_function_utils import (
  File "/Users/corey.zumar/mlflow/mlflow/gateway/uc_function_utils.py", line 9, in <module>
    from databricks.sdk import WorkspaceClient
ModuleNotFoundError: No module named 'databricks'
```

Reply:
Also, seems like this change will mean the `databricks.sdk` is required to run the MLflow Deployments server in general, which would be a significant issue for OSS users. Can we fix that?

Reply:
Can we also tell users to install the OpenAI Python SDK in the README?

Reply:
made `databricks.sdk` optional and added instructions on README.